src/commands/startStreamingLogs.ts

```diff
@@ -5,27 +5,32 @@
 
 import * as vscode from 'vscode';
 import * as appservice from 'vscode-azureappservice';
+import { LogStreamTreeItem } from '../explorer/LogStreamTreeItem';
 import { SiteTreeItem } from "../explorer/SiteTreeItem";
 import { WebAppTreeItem } from '../explorer/WebAppTreeItem';
 import { ext } from '../extensionVariables';
 import { enableFileLogging } from './enableFileLogging';
 
-export async function startStreamingLogs(node?: SiteTreeItem): Promise<void> {
+export async function startStreamingLogs(node?: SiteTreeItem | LogStreamTreeItem): Promise<void> {
+    let siteTreeItem: SiteTreeItem;
+
     if (!node) {
-        node = <WebAppTreeItem>await ext.tree.showTreeItemPicker(WebAppTreeItem.contextValue);
+        siteTreeItem = <WebAppTreeItem>await ext.tree.showTreeItemPicker(WebAppTreeItem.contextValue);
+    } else if (node instanceof LogStreamTreeItem) {
+        siteTreeItem = <SiteTreeItem>node.parent.parent;
+    } else {
+        siteTreeItem = <SiteTreeItem>node;
```

Comment:
nit: I don't think this cast is necessary. Typescript should know based on the other cases that this is SiteTreeItem